src/pages/home/Home.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components";
+import Header from "../../components/header/Header";
+import Hero from "../../components/hero/Hero";
+import Container from "../../components/styled-component/Container";
```

Comment:
Suggestion- Use absolute path instead of relative path as it increases the code readability.